auslib/web/views/client.py

```diff
@@ -51,9 +51,13 @@ def getQueryFromURL(self, url):
         return query
 
     def get(self, **url):
-        query = self.getQueryFromURL(url)
-        self.log.debug("Got query: %s", query)
-        release, update_type = AUS.evaluateRules(query)
+        try:
+            query = self.getQueryFromURL(url)
+            self.log.debug("Got query: %s", query)
+            release, update_type = AUS.evaluateRules(query)
+        except KeyError:
+            release = None
+
```

Comment:
Hm, I can't reproduce this....all of tests still pass even if I remove this block. Are you still getting an error with the latest code without it?